src/API/scenarios/scenario.controller.js

```diff
@@ -40,8 +40,9 @@ function update(req, res, next) {
 
 function remove(req, res, next) {
     req.scenario.removeAsync()
-        .then(() => {
-            res.end();
+        .then(sensor => {
+            res.json(sensor);
+            return next();
```

Comment:
afaik, we don`t have any middlewares, which need to be executed after delete, so this code is equivalent for now